examples/superdense_coding.py

```diff
@@ -0,0 +1,63 @@
+"""Superdense Coding.
+Superdense Coding is a method to transmit two classical bits of information
+from a sender to a receiver by sending only one qubit. This is accomplished
+by pre-sharing an entangled qubit.
+
+The follwoing example sets qubit 0 to 0 and qubit 1 to 1 and are output in
```

Comment:
Typo: c/follwoing/following

Reply:
Thanks so much for your thoughts here - I really appreciate it!

I just updated the code for this typo as well as the commentary point.